dashboard/app/models/levels/level.rb

```diff
@@ -694,6 +694,7 @@ def clone_with_name(new_name, editor_experiment: nil)
     # specify :published to make should_write_custom_level_file? return true
     level_params = {name: new_name, parent_level_id: id, published: true}
     level_params[:editor_experiment] = editor_experiment if editor_experiment
+    level_params[:audit_log] = '[]'
```

Comment:
Would it be useful to add something like `"cloned from #{name}"` here?

Reply:
great idea!

![Screen Shot 2021-07-15 at 1 29 12 PM](https://user-images.githubusercontent.com/8001765/125853569-2ecd39df-8014-4cdb-8c96-fccdd97db7ae.png)